internal/pkg/manifest/env_test.go

```diff
@@ -873,3 +924,63 @@ func TestEnvironmentCDNConfig_CDNEnabled(t *testing.T) {
 		})
 	}
 }
+
+func TestEnvironmentAccessLogs_ELBAccessLogs(t *testing.T) {
+	testCases := map[string]struct {
+		in            EnvironmentConfig
+		wantedFlag    bool
+		wantedConfigs *ELBAccessLogsArgs
+	}{
+		"enabled via bool": {
+			in: EnvironmentConfig{
+				HTTPConfig: EnvironmentHTTPConfig{
+					Public: PublicHTTPConfig{
+						ELBAccessLogs: ELBAccessLogsArgsOrBool{
+							Enabled: aws.Bool(true),
+						},
+					},
+				},
+			},
+			wantedFlag:    true,
+			wantedConfigs: nil,
+		},
+		"disabled via bool": {
+			in: EnvironmentConfig{
+				HTTPConfig: EnvironmentHTTPConfig{
+					Public: PublicHTTPConfig{
+						ELBAccessLogs: ELBAccessLogsArgsOrBool{},
```

Comment:
nit: is this test intended to be the following?
```suggestion
						ELBAccessLogs: ELBAccessLogsArgsOrBool{
						     Enabled: aws.Bool(false)
						},
```

Reply:
Actually it is for both if they give empty config or sets it to false. Both the configs will disable access logs.